### ISSUE TITLE
Add support of "interface direct" on firewalld

### ISSUE THREAD

**Issue by ansibot** [ansible/ansible#29193]
From @akram on 2015-01-20T11:07:08Z
##### ISSUE TYPE

Feature Idea
##### COMPONENT NAME

firewalld module
##### ANSIBLE VERSION

N/A
##### SUMMARY

hi guys,

that would be nice if the firewalld module could support the "interface direct" configuration, allowing us to run this kind of rules:
 firewall-cmd --permanent --direct --add-chain ipv4 filter FORWARD -i krb0  -o krb0 -j ACCEPT 

Copied from original issue: ansible/ansible-modules-extras#209

**Comment by ansibot** [comment]
From @akram on 2015-01-20T11:07:08Z

@akram Thanks for submitting this feature idea.

@maxamillion Pinging you to bring this feature request to your attention. Please note that you are under no particular obligation to add this feature.

We encourage all Ansible community members to submit pull requests to satisfy feature requests. Consult the module guidelines (http://docs.ansible.com/ansible/developing_modules.html) for more information.

**Comment by ansibot** [comment]
From @akram on 2015-01-20T11:07:08Z

[module: system/firewalld.py]

**Comment by ansibot** [comment]
From @akram on 2015-01-20T11:07:08Z

Yeah, I actually looked at this previously and it's something I would like to do. I'll see if I can't find some time to add it in the not too distant future.

**Comment by ansibot** [comment]
From @akram on 2015-01-20T11:07:08Z

@maxamillion, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
From @akram on 2015-01-20T11:07:08Z

@maxamillion, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
From @akram on 2015-01-20T11:07:08Z

@maxamillion, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by hdhoang** [comment]
duplicate of #21439

**Comment by ansibot** [comment]
Thank you very much for your interest in Ansible. Ansible has migrated much of the content into separate repositories to allow for more rapid, independent development. We are closing this issue/PR because this content has been moved to one or more collection repositories.

* lib/ansible/modules/system/firewalld.py -> https://galaxy.ansible.com/ansible/posix

For further information, please see:
https://github.com/ansible/ansibullbot/blob/master/docs/collection_migration.md
